tensorflow/core/common_runtime/gpu/pool_allocator.h

```diff
@@ -173,7 +173,11 @@ class BasicCPUAllocator : public SubAllocator {
   void* Alloc(size_t alignment, size_t num_bytes) override {
     return port::aligned_malloc(num_bytes, alignment);
   }
+#if defined(PLATFORM_WINDOWS)
```

Comment:
there should be a port::aligned_free which is valid for all cases. ASAN or valgrind should be able to flag that.